src/icon/index.css

```diff
@@ -210,6 +210,11 @@
   width: var(--spectrum-icon-doublegripper-width);
   height: var(--spectrum-icon-doublegripper-height);
 }
+.spectrum-UIIcon-FolderBreadcrumb {
+  /* Hacky: Use star vars for now */
```

Comment:
@NateBaldwinDesign I can use the real DNA variables once https://git.corp.adobe.com/Spectrum/spectrum-dna/pull/273/files is merged